lib/bots/versionbot.ts

```diff
@@ -425,16 +450,23 @@ export class VersionBot extends GithubBot.GithubBot {
             if (!statusesPassed) {
                 throw new Error(`At least one status check has failed; ${process.env.VERSIONBOT_NAME} will not ` +
                     'proceed to update this PR unless forced by re-applying the ' +
-                    '`procbots/versionbot/ready-to-merge` label');
+                    `\`${MergeLabel}\` label`);
             }
 
             // Ensure we've not already committed. If we have, we don't wish to do so again.
-            return this.versionBotCommits(prInfo);
+            return this.getVersionBotCommits(prInfo);
         }).then((commitMessage: string | null) => {
             if (commitMessage) {
                 throw new Error(`alreadyCommitted`);
             }
 
+            // If this was a labeling action and there's a config, check to see if there's a maintainers
+            // list and ensure the labeler was on it.
```

Comment:
This comment talks about "and there's a config" but you're no longer doing that check here.